src/http/nodegen/middleware/handleHttpException.ts

```diff
@@ -4,19 +4,38 @@ import { HttpException } from '../errors';
 import NodegenRequest from '../interfaces/NodegenRequest';
 import config from '@/config';
 
+export interface HttpExceptionOptions {
+  errorHook?: (error: any) => void,
+  errorLogger?: (error: any) => void,
+}
+
 /**
  * Http Exception handler
  */
-export default () => (err: HttpException, req: NodegenRequest, res: express.Response, next: express.NextFunction) => {
-  if (!(err instanceof HttpException)) {
-    err = createHttpExceptionFromErr(err);
+export default (options: HttpExceptionOptions = {}) => {
+  let errorLogger = (error: HttpException) => {
+    console.error(error.stack || error);
+  };
+
+  if (options.errorLogger && typeof options.errorLogger === 'function') {
```

Comment:
`typeof options?.errorLogger === 'function'` would suffice here

Reply:
:D PR is actually forked from your PR.. without the ? arg is can be made simpler yet